component-maps.yml

```diff
@@ -218,13 +218,6 @@ git:
     - mender-auditlogs
     release_component: true
 
-  mtls-ambassador:
-    docker_image:
-    - mtls-ambassador
-    docker_container:
-    - mtls-ambassador
-    release_component: true
```

Comment:
Please keep the component and just set `release_component` to `false` :+1:

Reply:
thanks amigo for early and unasked for review :* you always help

Reply:
fixed.